Polygon.Net.Tests/FunctionalTests/ReferenceApiTests.cs

```diff
@@ -44,18 +44,6 @@ public async Task GetTickerDetailsWithExpansionSucceedsAsync()
             Assert.IsNotNull(tickerDetailsResponse.Results.PhoneNumber);
         }
 
-        [TestMethod]

```

Comment:
It's always concerning to see a test get deleted. However this ticker no longer has a null Cik. We could look for another ticker with a null Cik but I don't think this test is that important. Also, the ticker that we do find may end up getting a Cik in the future like this one did.